tests/e2e/README.md

```diff
@@ -17,20 +17,31 @@ Run `make test-e2e` to execute the tests.
 | `project create` | Y |
 | `project get` | Y |
 | `project delete` | Y |
+| `project list` | N |
+| `project set` | N |
 | `app create`  | Y  |
 | `app get`  | Y  |
 | `app set` | Y |
 | `app delete` | Y |
+| `app describe` | N |
 | `app list` | Y |
-| `catalog` | N |
+| `catalog list` | Y |
+| `catalog search` | N |
+| `component get` | Y |
+| `component set` | Y |
 | `create --git` | N |
 | `create --local` | Y |
+| `create --binary` | N |
+| `delete` | N |
+| `describe` | N |
+| `link` | Y |
 | `list` | Y |
 | `push` | Y |
```

Comment:
are we covering push for every component type (--git/--local/--binary)?

Reply:
Will split and update this bit.